[ANGLE] Update ANGLE to 2023-11-06 (0584ffd8fbdd1e58dffceecea77c32989751def8)

djg opened the pull request:
#### e488bba9e00016e978951489976d5a13e1993ceb
<pre>
[ANGLE] Update ANGLE to 2023-11-06 (0584ffd8fbdd1e58dffceecea77c32989751def8)
<a href="https://bugs.webkit.org/show_bug.cgi?id=264312">https://bugs.webkit.org/show_bug.cgi?id=264312</a>
<a href="https://rdar.apple.com/114088465">rdar://114088465</a>

Reviewed by Kimmo Kinnunen

Contains upstream commits:
git log --oneline 06819324b129363163068c8f7ae8396bf83b4ec5..0584ffd8fbdd1e58dffceecea77c32989751def8 --pretty=%h %s
0584ffd8fb Unsuppress VUID-VkGraphicsPipelineCreateInfo-pStages-00739
c114fc265b Reland: Verify shader link correctness after looking up in program cache
5a71be901c D3D: Make non-constant loop indexing limitation a feature flag
2a433353ef Skip unnecessary default blend equation validation
e7ce481fe5 Consolidate parallel compilation in front-end
32f209b83f Vulkan: Fix __samplerExternal2DY2YEXTtnd combined with swizzle
8b4901d044 Avoid GLenum conversion in BlendStateExt blend and equation
ad4617f5fe Metal: Fix cross-compile build
34e5620b33 Metal: Reuse UBO conversion buffer only if size matches
eebf069c1d Roll vulkan-deps from 8c0916849657 to c4f7658f287b (6 revisions)
0e42512eed Roll Chromium from ab9dfa25fd4d to a63a2e689083 (785 revisions)
ae3af8ee0c Manual roll VK-GL-CTS from 7d738783bf28 to b127977ddad4 (29 revisions)
146a01ea1d Vulkan: Do not pad pipelineCacheData with zeroes
f441e3ee4f Metal: Fix missing mix(float,float,bool) overload
bfc764c553 Metal: Optimize vertex conversion using compute shader.
0a8a7c2bc4 Vulkan: Remove rx::WaitableCompressEvent class
59f7467a30 Metal: Add missing ScalarMatrix ops
eaae3b4b01 Roll vulkan-deps from c604e04c8136 to 8c0916849657 (7 revisions)
a3095f0cd9 Roll Chromium from c02a3bfd314f to ab9dfa25fd4d (687 revisions)
9f9c35f347 Metal: Don&apos;t map index buffer in drawArraysProvokingVertexImpl
67222ef476 Reland &quot;Vulkan: merge client buffer data.&quot;
221117abfb Tests: Add The Sims Mobile trace
4cc9e0afa0 Capture Tests: Fix unintentional gen-on-bind usage
6a879893fc Roll vulkan-deps from 54bfabb1adb0 to c604e04c8136 (9 revisions)
fbebf772ef Roll Chromium from c274ac05ad19 to c02a3bfd314f (535 revisions)
053aa9155a Vulkan: Disable VK_EXT_graphics_pipeline_library on RADV
918028a253 Update bot detection in capture_replay_tests
9a5d75de4b Vulkan: Fix incompatible redefinition of cube faces
3e54582615 Roll vulkan-deps from e55b4f78bcf3 to 54bfabb1adb0 (11 revisions)
f71a5bade9 Roll Chromium from 58d756134d9b to c274ac05ad19 (1331 revisions)
4681785688 Revert &quot;Vulkan: merge client buffer data.&quot;
4066fddda8 Add flag to move ANGLE binaries to /system/${LIB}.
563569acfc Vulkan: merge client buffer data.
c0c2067375 Remove expections for some AHB tests which are now fixed
b660767216 Test sampling from a texture with renderable levels below base
36f4199756 D3D11: CopySubTexture: Handle sRGB source with D3D11
2ee06400a6 Vulkan: fix dynamic buffer alignment issue.
c6dc00d213 Manual roll vulkan-deps from 69081d0e32f7 to e55b4f78bcf3 (110 revisions)
118a9b142e Test sampling from a texture with extra renderable levels
907ccf8419 Vulkan: Stop compiling vulkan validation layers on Android
f4a472332c DEPS: update reclient
85549ffa89 Roll SwiftShader from d9ec9befba05 to 5c95af798ba4 (1 revision)
8e8b4dc940 Aosp builds -O2 instead of -Os
c0f94afb70 Vulkan: Limit ImageTestES3.RGBXAHBImportMultipleLayers to 1 layer
471b50407d Vulkan: Only enable DS dynamic state if there is DS attachment.
d67aafacd3 Keep default_optimization on Fuchsia
5fa65ec81e ImageTest: check support before test cubemap
e519da9a27 Tests: Add Streets of Rage 4 trace
b48d9daf00 Capture/Replay: Gen gen-on-bind resources on replay
bcd331f15f ImageTest: Allow writeAHBData to fail
53c5b14506 Capture/Replay: Restore texture bindings on reset
acb0ae0931 Vulkan: Swizzle YUV output from GL component order to vulkan
3b555e8da6 GL: Add emulateClipOrigin workaround
b56a72045a Roll Chromium from 63a05c417b61 to 58d756134d9b (825 revisions)
95fe7e4704 [Android] Update NDK version check for hardware buffer.
a7882c416c Skip InstancingTestES3.LargeDivisor on Linux ASAN

* Source/ThirdParty/ANGLE/.gitmodules:
* Source/ThirdParty/ANGLE/ANGLE.plist:
* Source/ThirdParty/ANGLE/DEPS:
* Source/ThirdParty/ANGLE/Metal.cmake:
* Source/ThirdParty/ANGLE/WebKit/ANGLEShaderProgramVersion.h:
* Source/ThirdParty/ANGLE/WebKit/angle_commit.h:
* Source/ThirdParty/ANGLE/changes.diff:
* Source/ThirdParty/ANGLE/gni/angle.gni:
* Source/ThirdParty/ANGLE/include/GLSLANG/ShaderLang.h:
* Source/ThirdParty/ANGLE/include/platform/autogen/FeaturesD3D_autogen.h:
* Source/ThirdParty/ANGLE/include/platform/autogen/FeaturesGL_autogen.h:
* Source/ThirdParty/ANGLE/include/platform/autogen/FrontendFeatures_autogen.h:
* Source/ThirdParty/ANGLE/include/platform/d3d_features.json:
* Source/ThirdParty/ANGLE/include/platform/frontend_features.json:
* Source/ThirdParty/ANGLE/include/platform/gl_features.json:
* Source/ThirdParty/ANGLE/scripts/code_generation_hashes/D3D11_format.json:
* Source/ThirdParty/ANGLE/scripts/code_generation_hashes/GL_CTS_(dEQP)_build_files.json:
* Source/ThirdParty/ANGLE/scripts/code_generation_hashes/Vulkan_mandatory_format_support_table.json:
* Source/ThirdParty/ANGLE/scripts/code_generation_hashes/restricted_traces.json:
* Source/ThirdParty/ANGLE/scripts/generate_android_bp.py:
(library_target_to_blueprint):
(get_angle_on_system_flag_config):
(main):
* Source/ThirdParty/ANGLE/src/compiler/translator/Compiler.cpp:
(sh::TCompiler::checkAndSimplifyAST):
* Source/ThirdParty/ANGLE/src/compiler/translator/glsl/TranslatorESSL.cpp:
(sh::TranslatorESSL::translate):
* Source/ThirdParty/ANGLE/src/compiler/translator/msl/EmitMetal.cpp:
(GetOperatorString):
* Source/ThirdParty/ANGLE/src/compiler/translator/msl/ProgramPrelude.cpp:
* Source/ThirdParty/ANGLE/src/compiler/translator/spirv/TranslatorSPIRV.cpp:
(sh::TranslatorSPIRV::translateImpl):
* Source/ThirdParty/ANGLE/src/compiler/translator/tree_ops/spirv/ReswizzleYUVOps.cpp:
(sh::ReswizzleYUVOps):
* Source/ThirdParty/ANGLE/src/compiler/translator/tree_ops/spirv/ReswizzleYUVOps.h:
* Source/ThirdParty/ANGLE/src/libANGLE/BlendStateExt_unittest.cpp:
(angle::TEST):
* Source/ThirdParty/ANGLE/src/libANGLE/Caps.h:
* Source/ThirdParty/ANGLE/src/libANGLE/MemoryProgramCache.cpp:
(gl::MemoryProgramCache::ComputeHash):
* Source/ThirdParty/ANGLE/src/libANGLE/Program.cpp:
(gl::Program::linkImpl):
* Source/ThirdParty/ANGLE/src/libANGLE/Shader.cpp:
(gl::Shader::Shader):
(gl::Shader::compile):
(gl::Shader::resolveCompile):
(gl::Shader::isCompleted):
* Source/ThirdParty/ANGLE/src/libANGLE/Shader.h:
* Source/ThirdParty/ANGLE/src/libANGLE/State.cpp:
(gl::PrivateState::getIntegerv const):
(gl::PrivateState::getIntegeri_v const):
* Source/ThirdParty/ANGLE/src/libANGLE/angletypes.cpp:
(gl::BlendStateExt::getEquationColorIndexed const): Deleted.
(gl::BlendStateExt::getEquationAlphaIndexed const): Deleted.
(gl::BlendStateExt::getSrcColorIndexed const): Deleted.
(gl::BlendStateExt::getDstColorIndexed const): Deleted.
(gl::BlendStateExt::getSrcAlphaIndexed const): Deleted.
(gl::BlendStateExt::getDstAlphaIndexed const): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/angletypes.h:
* Source/ThirdParty/ANGLE/src/libANGLE/capture/FrameCapture.cpp:
(angle::FrameCaptureShared::maybeCapturePreCallUpdates):
(angle::FrameCaptureShared::runMidExecutionCapture):
* Source/ThirdParty/ANGLE/src/libANGLE/capture/FrameCapture.h:
* Source/ThirdParty/ANGLE/src/libANGLE/capture/serialize.cpp:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/ShaderImpl.cpp:
(rx::ShaderTranslateTask::translate):
(rx::WaitableCompileEvent::WaitableCompileEvent): Deleted.
(rx::WaitableCompileEvent::~WaitableCompileEvent): Deleted.
(rx::WaitableCompileEvent::wait): Deleted.
(rx::WaitableCompileEvent::isReady): Deleted.
(rx::WaitableCompileEvent::getInfoLog): Deleted.
(rx::TranslateTask::TranslateTask): Deleted.
(): Deleted.
(rx::TranslateTask::getResult): Deleted.
(rx::TranslateTask::getHandle): Deleted.
(rx::ShaderImpl::compileImpl): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/ShaderImpl.h:
(rx::ShaderTranslateTask::postTranslate):
(rx::ShaderTranslateTask::isCompilingInternally):
(rx::ShaderTranslateTask::getResult):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/ShaderD3D.cpp:
(rx::ShaderD3D::compile):
(rx::TranslateTaskD3D::TranslateTaskD3D): Deleted.
(rx::TranslateTaskD3D::getResult): Deleted.
(rx::GetUniformRegisterMap): Deleted.
(rx::GetSlowCompilingUniformBlockSet): Deleted.
(rx::GetUsedImage2DFunctionNames): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/ShaderD3D.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/RenderStateCache.cpp:
(rx::RenderStateCache::GetBlendStateKey):
(rx::RenderStateCache::getBlendState):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp:
(rx::Renderer11::copyTexture):
(rx::Renderer11::generateMipmapUsingD3D):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/TextureStorage11.cpp:
(rx::TextureStorage11::SamplerKey::SamplerKey):
(rx::TextureStorage11::getSRVForSampler):
(rx::TextureStorage11::getCachedOrCreateSRVForSampler):
(rx::TextureStorage11::getSRVLevels):
(rx::TextureStorage11::requiresTypelessTextureFormat const):
(rx::TextureStorage11_2D::ensureTextureExists):
(rx::TextureStorage11_Cube::ensureTextureExists):
(rx::TextureStorage11_3D::getResource):
(rx::TextureStorage11_2DArray::getResource):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/TextureStorage11.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/formatutils11.cpp:
(rx::d3d11::GetYUVPlaneFormat):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/gen_texture_format_table.py:
(get_blit_srv_format):
(json_to_table_data):
(get_stencil_srv_format): Deleted.
(get_typeless_format): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp:
(rx::d3d11_gl::GenerateCaps):
(rx::d3d11::InitializeFeatures):
(rx::d3d11::InitializeFrontendFeatures):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/texture_format_data.json:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/texture_format_table.h:
(rx::d3d11::Format::Format):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/texture_format_table_autogen.cpp:
(rx::d3d11::Format::Get):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d9/renderer9_utils.cpp:
(rx::d3d9_gl::GenerateCaps):
(rx::d3d9::InitializeFeatures):
(rx::d3d9::InitializeFrontendFeatures):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/ProgramExecutableGL.cpp:
(rx::ProgramExecutableGL::ProgramExecutableGL):
(rx::ProgramExecutableGL::reset):
(rx::ProgramExecutableGL::postLink):
(rx::ProgramExecutableGL::updateEmulatedClipOrigin const):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/ProgramExecutableGL.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/ShaderGL.cpp:
(rx::ShaderGL::ShaderGL):
(rx::ShaderGL::compile):
(rx::ShaderGL::compileAndCheckShader): Deleted.
(rx::ShaderGL::compileShader): Deleted.
(rx::ShaderGL::checkShader): Deleted.
(rx::ShaderGL::peekCompletion): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/ShaderGL.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/StateManagerGL.cpp:
(rx::StateManagerGL::setClipControlWithEmulatedClipOrigin):
(rx::StateManagerGL::setBlendFuncs):
(rx::StateManagerGL::setBlendEquations):
(rx::StateManagerGL::syncState):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/StateManagerGL.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/renderergl_utils.cpp:
(rx::nativegl_gl::InitializeFeatures):
(rx::nativegl_gl::InitializeFrontendFeatures):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/BufferMtl.mm:
(rx::BufferMtl::getUniformConversionBuffer):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ContextMtl.mm:
(rx::ContextMtl::drawArraysProvokingVertexImpl):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/DisplayMtl.mm:
(rx::DisplayMtl::initializeFrontendFeatures const):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ProgramExecutableMtl.mm:
(rx::ProgramExecutableMtl::legalizeUniformBufferOffsets):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ShaderMtl.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ShaderMtl.mm:
(rx::ShaderMtl::ShaderMtl):
(rx::ShaderMtl::~ShaderMtl):
(rx::ShaderMtl::compile):
(rx::TranslateTask::TranslateTask): Deleted.
(rx::TranslateTask::getResult): Deleted.
(rx::TranslateTask::getHandle): Deleted.
(rx::ShaderMtl::compileImplMtl): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/VertexArrayMtl.mm:
(rx::VertexArrayMtl::convertVertexBuffer):
(rx::VertexArrayMtl::convertVertexBufferGPU):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/metal_backend.gni:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_render_utils.mm:
(rx::mtl::VertexFormatConversionUtils::convertVertexFormatToFloatCS):
(rx::mtl::VertexFormatConversionUtils::expandVertexFormatComponentsCS):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_utils.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_utils.mm:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/null/ShaderNULL.cpp:
(rx::ShaderNULL::compile):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/null/ShaderNULL.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ContextVk.cpp:
(rx::ContextVk::initialize):
(rx::ContextVk::setupDraw):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ContextVk.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/RendererVk.cpp:
(rx::RendererVk::initFeatures):
(rx::RendererVk::initializeFrontendFeatures const):
(rx::RendererVk::syncPipelineCacheVk):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/RendererVk.h:
(rx::WaitableCompressEvent::WaitableCompressEvent): Deleted.
(rx::WaitableCompressEvent::~WaitableCompressEvent): Deleted.
(rx::WaitableCompressEvent::wait): Deleted.
(rx::WaitableCompressEvent::isReady): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ShaderVk.cpp:
(rx::ShaderVk::compile):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ShaderVk.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/TextureVk.cpp:
(rx::TextureVk::shouldUpdateBeStaged const):
(rx::TextureVk::setStorageMultisample):
(rx::TextureVk::releaseAndDeleteImageAndViews):
(rx::TextureVk::redefineLevel):
(rx::TextureVk::generateMipmapsWithCPU):
(rx::TextureVk::copyAndStageImageData):
(rx::TextureVk::reinitImageAsRenderable):
(rx::TextureVk::respecifyImageStorage):
(rx::TextureVk::ensureImageInitialized):
(rx::TextureVk::prepareForGenerateMipmap):
(rx::TextureVk::respecifyImageStorageIfNecessary):
(rx::TextureVk::releaseImage):
(rx::TextureVk::stageSelfAsSubresourceUpdates):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/TextureVk.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/VertexArrayVk.cpp:
(rx::VertexArrayVk::syncDirtyAttrib):
(rx::VertexArrayVk::mergeClientAttribsRange const):
(rx::VertexArrayVk::updateStreamedAttribs):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/VertexArrayVk.h:
(rx::AttributeRange::AttributeRange):
(rx::operator&lt;):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/vk_cache_utils.cpp:
(rx::vk::RenderPassDesc::hasDepthAttachment const):
(rx::vk::RenderPassDesc::hasStencilAttachment const):
(rx::vk::GraphicsPipelineDesc::initializePipelineShadersState const):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/vk_cache_utils.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/vk_helpers.cpp:
(rx::vk::ImageHelper::stageSelfAsSubresourceUpdates):
(rx::vk::ImageHelper::flushStagedUpdates):
(rx::vk::ImageHelper::removeSupersededUpdates):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/vk_helpers.h:
* Source/ThirdParty/ANGLE/src/libANGLE/validationES.cpp:
(gl::err::ValidateProgramDrawAdvancedBlendState):
* Source/ThirdParty/ANGLE/src/libANGLE/validationESEXT.cpp:
* Source/ThirdParty/ANGLE/src/tests/angle_end2end_tests_expectations.txt:
* Source/ThirdParty/ANGLE/src/tests/capture_replay_tests.py:
(main):
* Source/ThirdParty/ANGLE/src/tests/capture_tests/expected/CapturedTest_MultiFrame_ES3_Vulkan.json:
* Source/ThirdParty/ANGLE/src/tests/capture_tests/expected/CapturedTest_MultiFrame_ES3_Vulkan_001.cpp:
(ResetReplay):
* Source/ThirdParty/ANGLE/src/tests/compiler_tests/EXT_YUV_target_test.cpp:
* Source/ThirdParty/ANGLE/src/tests/deqp_support/deqp_khr_gles2_test_expectations.txt:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/BlitFramebufferANGLETest.cpp:
(angle::TEST_P):
* Source/ThirdParty/ANGLE/src/tests/gl_tests/ClipControlTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/DepthStencilFormatsTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/GLSLTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/ImageTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/ObjectAllocationTest.cpp:
(angle::TEST_P):
* Source/ThirdParty/ANGLE/src/tests/gl_tests/TextureTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/UniformBufferTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/VertexAttributeTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/restricted_traces/restricted_traces.json:
* Source/ThirdParty/ANGLE/src/tests/test_utils/ShaderExtensionTest.h:
(sh::ShaderExtensionTest::TestShaderCompile):
* Source/ThirdParty/ANGLE/util/autogen/angle_features_autogen.cpp:
* Source/ThirdParty/ANGLE/util/autogen/angle_features_autogen.h:
* Source/ThirdParty/ANGLE/util/capture/trace_fixture.cpp:
(SetFramebufferID):
(SetBufferID):
(SetRenderbufferID):
(SetTextureID):
(SetResourceID): Deleted.

Canonical link: <a href="https://commits.webkit.org/270363@main">https://commits.webkit.org/270363@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4d36164ddedbb1cd400a1789bd270c98ef7d5572

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/25283 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/3824 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/26538 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/27394 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/23193 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/5541 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/1257 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/23401 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/25525 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/2834 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/21821 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/27971 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/2514 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/22761 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/28862 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/23063 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/23113 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/26696 "Passed tests") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/2468 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/753 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/3822 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/6058 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/2909 "Built successfully") | | [⏳ 🛠 jsc-mips ](https://ews-build.webkit.org/#/builders/JSC-MIPSEL-32bits-Build-EWS "Waiting in queue, processing has not started yet") | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/45/builds/2803 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [⏳ 🧪 jsc-mips-tests ](https://ews-build.webkit.org/#/builders/JSC-MIPSEL-32bits-Build-EWS "Waiting in queue, processing has not started yet") | 
<!--EWS-Status-Bubble-End-->

kkinnunen-apple commented on Source/ThirdParty/ANGLE/changes.diff:
```diff
@@ -1,3 +1,139 @@
+diff --git a/.gitmodules b/.gitmodules
+index 6a5d4a9f2b638ddb9f3d59d01b37ccb4c76adcab..cc30569e4ac565aa5f77c42d8d029e9780fd785d 100644
```
you need to copy the .gitmodules manuallly from the upstream revision you¨re merging and regenerate the diffs

djg replied:
Thanks for the heads up.

djg committed:
[ANGLE] Update ANGLE to 2023-11-06 (0584ffd8fbdd1e58dffceecea77c32989751def8)
https://bugs.webkit.org/show_bug.cgi?id=264312
rdar://114088465

Reviewed by Kimmo Kinnunen

Contains upstream commits:
git log --oneline 06819324b129363163068c8f7ae8396bf83b4ec5..0584ffd8fbdd1e58dffceecea77c32989751def8 --pretty=%h %s
0584ffd8fb Unsuppress VUID-VkGraphicsPipelineCreateInfo-pStages-00739
c114fc265b Reland: Verify shader link correctness after looking up in program cache
5a71be901c D3D: Make non-constant loop indexing limitation a feature flag
2a433353ef Skip unnecessary default blend equation validation
e7ce481fe5 Consolidate parallel compilation in front-end
32f209b83f Vulkan: Fix __samplerExternal2DY2YEXTtnd combined with swizzle
8b4901d044 Avoid GLenum conversion in BlendStateExt blend and equation
ad4617f5fe Metal: Fix cross-compile build
34e5620b33 Metal: Reuse UBO conversion buffer only if size matches
eebf069c1d Roll vulkan-deps from 8c0916849657 to c4f7658f287b (6 revisions)
0e42512eed Roll Chromium from ab9dfa25fd4d to a63a2e689083 (785 revisions)
ae3af8ee0c Manual roll VK-GL-CTS from 7d738783bf28 to b127977ddad4 (29 revisions)
146a01ea1d Vulkan: Do not pad pipelineCacheData with zeroes
f441e3ee4f Metal: Fix missing mix(float,float,bool) overload
bfc764c553 Metal: Optimize vertex conversion using compute shader.
0a8a7c2bc4 Vulkan: Remove rx::WaitableCompressEvent class
59f7467a30 Metal: Add missing ScalarMatrix ops
eaae3b4b01 Roll vulkan-deps from c604e04c8136 to 8c0916849657 (7 revisions)
a3095f0cd9 Roll Chromium from c02a3bfd314f to ab9dfa25fd4d (687 revisions)
9f9c35f347 Metal: Don't map index buffer in drawArraysProvokingVertexImpl
67222ef476 Reland "Vulkan: merge client buffer data."
221117abfb Tests: Add The Sims Mobile trace
4cc9e0afa0 Capture Tests: Fix unintentional gen-on-bind usage
6a879893fc Roll vulkan-deps from 54bfabb1adb0 to c604e04c8136 (9 revisions)
fbebf772ef Roll Chromium from c274ac05ad19 to c02a3bfd314f (535 revisions)
053aa9155a Vulkan: Disable VK_EXT_graphics_pipeline_library on RADV
918028a253 Update bot detection in capture_replay_tests
9a5d75de4b Vulkan: Fix incompatible redefinition of cube faces
3e54582615 Roll vulkan-deps from e55b4f78bcf3 to 54bfabb1adb0 (11 revisions)
f71a5bade9 Roll Chromium from 58d756134d9b to c274ac05ad19 (1331 revisions)
4681785688 Revert "Vulkan: merge client buffer data."
4066fddda8 Add flag to move ANGLE binaries to /system/${LIB}.
563569acfc Vulkan: merge client buffer data.
c0c2067375 Remove expections for some AHB tests which are now fixed
b660767216 Test sampling from a texture with renderable levels below base
36f4199756 D3D11: CopySubTexture: Handle sRGB source with D3D11
2ee06400a6 Vulkan: fix dynamic buffer alignment issue.
c6dc00d213 Manual roll vulkan-deps from 69081d0e32f7 to e55b4f78bcf3 (110 revisions)
118a9b142e Test sampling from a texture with extra renderable levels
907ccf8419 Vulkan: Stop compiling vulkan validation layers on Android
f4a472332c DEPS: update reclient
85549ffa89 Roll SwiftShader from d9ec9befba05 to 5c95af798ba4 (1 revision)
8e8b4dc940 Aosp builds -O2 instead of -Os
c0f94afb70 Vulkan: Limit ImageTestES3.RGBXAHBImportMultipleLayers to 1 layer
471b50407d Vulkan: Only enable DS dynamic state if there is DS attachment.
d67aafacd3 Keep default_optimization on Fuchsia
5fa65ec81e ImageTest: check support before test cubemap
e519da9a27 Tests: Add Streets of Rage 4 trace
b48d9daf00 Capture/Replay: Gen gen-on-bind resources on replay
bcd331f15f ImageTest: Allow writeAHBData to fail
53c5b14506 Capture/Replay: Restore texture bindings on reset
acb0ae0931 Vulkan: Swizzle YUV output from GL component order to vulkan
3b555e8da6 GL: Add emulateClipOrigin workaround
b56a72045a Roll Chromium from 63a05c417b61 to 58d756134d9b (825 revisions)
95fe7e4704 [Android] Update NDK version check for hardware buffer.
a7882c416c Skip InstancingTestES3.LargeDivisor on Linux ASAN

* Source/ThirdParty/ANGLE/.gitmodules:
* Source/ThirdParty/ANGLE/ANGLE.plist:
* Source/ThirdParty/ANGLE/DEPS:
* Source/ThirdParty/ANGLE/Metal.cmake:
* Source/ThirdParty/ANGLE/WebKit/ANGLEShaderProgramVersion.h:
* Source/ThirdParty/ANGLE/WebKit/angle_commit.h:
* Source/ThirdParty/ANGLE/changes.diff:
* Source/ThirdParty/ANGLE/gni/angle.gni:
* Source/ThirdParty/ANGLE/include/GLSLANG/ShaderLang.h:
* Source/ThirdParty/ANGLE/include/platform/autogen/FeaturesD3D_autogen.h:
* Source/ThirdParty/ANGLE/include/platform/autogen/FeaturesGL_autogen.h:
* Source/ThirdParty/ANGLE/include/platform/autogen/FrontendFeatures_autogen.h:
* Source/ThirdParty/ANGLE/include/platform/d3d_features.json:
* Source/ThirdParty/ANGLE/include/platform/frontend_features.json:
* Source/ThirdParty/ANGLE/include/platform/gl_features.json:
* Source/ThirdParty/ANGLE/scripts/code_generation_hashes/D3D11_format.json:
* Source/ThirdParty/ANGLE/scripts/code_generation_hashes/GL_CTS_(dEQP)_build_files.json:
* Source/ThirdParty/ANGLE/scripts/code_generation_hashes/Vulkan_mandatory_format_support_table.json:
* Source/ThirdParty/ANGLE/scripts/code_generation_hashes/restricted_traces.json:
* Source/ThirdParty/ANGLE/scripts/generate_android_bp.py:
(library_target_to_blueprint):
(get_angle_on_system_flag_config):
(main):
* Source/ThirdParty/ANGLE/src/compiler/translator/Compiler.cpp:
(sh::TCompiler::checkAndSimplifyAST):
* Source/ThirdParty/ANGLE/src/compiler/translator/glsl/TranslatorESSL.cpp:
(sh::TranslatorESSL::translate):
* Source/ThirdParty/ANGLE/src/compiler/translator/msl/EmitMetal.cpp:
(GetOperatorString):
* Source/ThirdParty/ANGLE/src/compiler/translator/msl/ProgramPrelude.cpp:
* Source/ThirdParty/ANGLE/src/compiler/translator/spirv/TranslatorSPIRV.cpp:
(sh::TranslatorSPIRV::translateImpl):
* Source/ThirdParty/ANGLE/src/compiler/translator/tree_ops/spirv/ReswizzleYUVOps.cpp:
(sh::ReswizzleYUVOps):
* Source/ThirdParty/ANGLE/src/compiler/translator/tree_ops/spirv/ReswizzleYUVOps.h:
* Source/ThirdParty/ANGLE/src/libANGLE/BlendStateExt_unittest.cpp:
(angle::TEST):
* Source/ThirdParty/ANGLE/src/libANGLE/Caps.h:
* Source/ThirdParty/ANGLE/src/libANGLE/MemoryProgramCache.cpp:
(gl::MemoryProgramCache::ComputeHash):
* Source/ThirdParty/ANGLE/src/libANGLE/Program.cpp:
(gl::Program::linkImpl):
* Source/ThirdParty/ANGLE/src/libANGLE/Shader.cpp:
(gl::Shader::Shader):
(gl::Shader::compile):
(gl::Shader::resolveCompile):
(gl::Shader::isCompleted):
* Source/ThirdParty/ANGLE/src/libANGLE/Shader.h:
* Source/ThirdParty/ANGLE/src/libANGLE/State.cpp:
(gl::PrivateState::getIntegerv const):
(gl::PrivateState::getIntegeri_v const):
* Source/ThirdParty/ANGLE/src/libANGLE/angletypes.cpp:
(gl::BlendStateExt::getEquationColorIndexed const): Deleted.
(gl::BlendStateExt::getEquationAlphaIndexed const): Deleted.
(gl::BlendStateExt::getSrcColorIndexed const): Deleted.
(gl::BlendStateExt::getDstColorIndexed const): Deleted.
(gl::BlendStateExt::getSrcAlphaIndexed const): Deleted.
(gl::BlendStateExt::getDstAlphaIndexed const): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/angletypes.h:
* Source/ThirdParty/ANGLE/src/libANGLE/capture/FrameCapture.cpp:
(angle::FrameCaptureShared::maybeCapturePreCallUpdates):
(angle::FrameCaptureShared::runMidExecutionCapture):
* Source/ThirdParty/ANGLE/src/libANGLE/capture/FrameCapture.h:
* Source/ThirdParty/ANGLE/src/libANGLE/capture/serialize.cpp:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/ShaderImpl.cpp:
(rx::ShaderTranslateTask::translate):
(rx::WaitableCompileEvent::WaitableCompileEvent): Deleted.
(rx::WaitableCompileEvent::~WaitableCompileEvent): Deleted.
(rx::WaitableCompileEvent::wait): Deleted.
(rx::WaitableCompileEvent::isReady): Deleted.
(rx::WaitableCompileEvent::getInfoLog): Deleted.
(rx::TranslateTask::TranslateTask): Deleted.
(): Deleted.
(rx::TranslateTask::getResult): Deleted.
(rx::TranslateTask::getHandle): Deleted.
(rx::ShaderImpl::compileImpl): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/ShaderImpl.h:
(rx::ShaderTranslateTask::postTranslate):
(rx::ShaderTranslateTask::isCompilingInternally):
(rx::ShaderTranslateTask::getResult):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/ShaderD3D.cpp:
(rx::ShaderD3D::compile):
(rx::TranslateTaskD3D::TranslateTaskD3D): Deleted.
(rx::TranslateTaskD3D::getResult): Deleted.
(rx::GetUniformRegisterMap): Deleted.
(rx::GetSlowCompilingUniformBlockSet): Deleted.
(rx::GetUsedImage2DFunctionNames): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/ShaderD3D.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/RenderStateCache.cpp:
(rx::RenderStateCache::GetBlendStateKey):
(rx::RenderStateCache::getBlendState):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp:
(rx::Renderer11::copyTexture):
(rx::Renderer11::generateMipmapUsingD3D):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/TextureStorage11.cpp:
(rx::TextureStorage11::SamplerKey::SamplerKey):
(rx::TextureStorage11::getSRVForSampler):
(rx::TextureStorage11::getCachedOrCreateSRVForSampler):
(rx::TextureStorage11::getSRVLevels):
(rx::TextureStorage11::requiresTypelessTextureFormat const):
(rx::TextureStorage11_2D::ensureTextureExists):
(rx::TextureStorage11_Cube::ensureTextureExists):
(rx::TextureStorage11_3D::getResource):
(rx::TextureStorage11_2DArray::getResource):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/TextureStorage11.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/formatutils11.cpp:
(rx::d3d11::GetYUVPlaneFormat):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/gen_texture_format_table.py:
(get_blit_srv_format):
(json_to_table_data):
(get_stencil_srv_format): Deleted.
(get_typeless_format): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp:
(rx::d3d11_gl::GenerateCaps):
(rx::d3d11::InitializeFeatures):
(rx::d3d11::InitializeFrontendFeatures):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/texture_format_data.json:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/texture_format_table.h:
(rx::d3d11::Format::Format):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d11/texture_format_table_autogen.cpp:
(rx::d3d11::Format::Get):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/d3d/d3d9/renderer9_utils.cpp:
(rx::d3d9_gl::GenerateCaps):
(rx::d3d9::InitializeFeatures):
(rx::d3d9::InitializeFrontendFeatures):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/ProgramExecutableGL.cpp:
(rx::ProgramExecutableGL::ProgramExecutableGL):
(rx::ProgramExecutableGL::reset):
(rx::ProgramExecutableGL::postLink):
(rx::ProgramExecutableGL::updateEmulatedClipOrigin const):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/ProgramExecutableGL.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/ShaderGL.cpp:
(rx::ShaderGL::ShaderGL):
(rx::ShaderGL::compile):
(rx::ShaderGL::compileAndCheckShader): Deleted.
(rx::ShaderGL::compileShader): Deleted.
(rx::ShaderGL::checkShader): Deleted.
(rx::ShaderGL::peekCompletion): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/ShaderGL.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/StateManagerGL.cpp:
(rx::StateManagerGL::setClipControlWithEmulatedClipOrigin):
(rx::StateManagerGL::setBlendFuncs):
(rx::StateManagerGL::setBlendEquations):
(rx::StateManagerGL::syncState):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/StateManagerGL.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/renderergl_utils.cpp:
(rx::nativegl_gl::InitializeFeatures):
(rx::nativegl_gl::InitializeFrontendFeatures):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/BufferMtl.mm:
(rx::BufferMtl::getUniformConversionBuffer):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ContextMtl.mm:
(rx::ContextMtl::drawArraysProvokingVertexImpl):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/DisplayMtl.mm:
(rx::DisplayMtl::initializeFrontendFeatures const):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ProgramExecutableMtl.mm:
(rx::ProgramExecutableMtl::legalizeUniformBufferOffsets):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ShaderMtl.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ShaderMtl.mm:
(rx::ShaderMtl::ShaderMtl):
(rx::ShaderMtl::~ShaderMtl):
(rx::ShaderMtl::compile):
(rx::TranslateTask::TranslateTask): Deleted.
(rx::TranslateTask::getResult): Deleted.
(rx::TranslateTask::getHandle): Deleted.
(rx::ShaderMtl::compileImplMtl): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/VertexArrayMtl.mm:
(rx::VertexArrayMtl::convertVertexBuffer):
(rx::VertexArrayMtl::convertVertexBufferGPU):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/metal_backend.gni:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_render_utils.mm:
(rx::mtl::VertexFormatConversionUtils::convertVertexFormatToFloatCS):
(rx::mtl::VertexFormatConversionUtils::expandVertexFormatComponentsCS):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_utils.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_utils.mm:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/null/ShaderNULL.cpp:
(rx::ShaderNULL::compile):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/null/ShaderNULL.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ContextVk.cpp:
(rx::ContextVk::initialize):
(rx::ContextVk::setupDraw):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ContextVk.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/RendererVk.cpp:
(rx::RendererVk::initFeatures):
(rx::RendererVk::initializeFrontendFeatures const):
(rx::RendererVk::syncPipelineCacheVk):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/RendererVk.h:
(rx::WaitableCompressEvent::WaitableCompressEvent): Deleted.
(rx::WaitableCompressEvent::~WaitableCompressEvent): Deleted.
(rx::WaitableCompressEvent::wait): Deleted.
(rx::WaitableCompressEvent::isReady): Deleted.
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ShaderVk.cpp:
(rx::ShaderVk::compile):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ShaderVk.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/TextureVk.cpp:
(rx::TextureVk::shouldUpdateBeStaged const):
(rx::TextureVk::setStorageMultisample):
(rx::TextureVk::releaseAndDeleteImageAndViews):
(rx::TextureVk::redefineLevel):
(rx::TextureVk::generateMipmapsWithCPU):
(rx::TextureVk::copyAndStageImageData):
(rx::TextureVk::reinitImageAsRenderable):
(rx::TextureVk::respecifyImageStorage):
(rx::TextureVk::ensureImageInitialized):
(rx::TextureVk::prepareForGenerateMipmap):
(rx::TextureVk::respecifyImageStorageIfNecessary):
(rx::TextureVk::releaseImage):
(rx::TextureVk::stageSelfAsSubresourceUpdates):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/TextureVk.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/VertexArrayVk.cpp:
(rx::VertexArrayVk::syncDirtyAttrib):
(rx::VertexArrayVk::mergeClientAttribsRange const):
(rx::VertexArrayVk::updateStreamedAttribs):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/VertexArrayVk.h:
(rx::AttributeRange::AttributeRange):
(rx::operator<):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/vk_cache_utils.cpp:
(rx::vk::RenderPassDesc::hasDepthAttachment const):
(rx::vk::RenderPassDesc::hasStencilAttachment const):
(rx::vk::GraphicsPipelineDesc::initializePipelineShadersState const):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/vk_cache_utils.h:
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/vk_helpers.cpp:
(rx::vk::ImageHelper::stageSelfAsSubresourceUpdates):
(rx::vk::ImageHelper::flushStagedUpdates):
(rx::vk::ImageHelper::removeSupersededUpdates):
* Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/vk_helpers.h:
* Source/ThirdParty/ANGLE/src/libANGLE/validationES.cpp:
(gl::err::ValidateProgramDrawAdvancedBlendState):
* Source/ThirdParty/ANGLE/src/libANGLE/validationESEXT.cpp:
* Source/ThirdParty/ANGLE/src/tests/angle_end2end_tests_expectations.txt:
* Source/ThirdParty/ANGLE/src/tests/capture_replay_tests.py:
(main):
* Source/ThirdParty/ANGLE/src/tests/capture_tests/expected/CapturedTest_MultiFrame_ES3_Vulkan.json:
* Source/ThirdParty/ANGLE/src/tests/capture_tests/expected/CapturedTest_MultiFrame_ES3_Vulkan_001.cpp:
(ResetReplay):
* Source/ThirdParty/ANGLE/src/tests/compiler_tests/EXT_YUV_target_test.cpp:
* Source/ThirdParty/ANGLE/src/tests/deqp_support/deqp_khr_gles2_test_expectations.txt:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/BlitFramebufferANGLETest.cpp:
(angle::TEST_P):
* Source/ThirdParty/ANGLE/src/tests/gl_tests/ClipControlTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/DepthStencilFormatsTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/GLSLTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/ImageTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/ObjectAllocationTest.cpp:
(angle::TEST_P):
* Source/ThirdParty/ANGLE/src/tests/gl_tests/TextureTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/UniformBufferTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/gl_tests/VertexAttributeTest.cpp:
* Source/ThirdParty/ANGLE/src/tests/restricted_traces/restricted_traces.json:
* Source/ThirdParty/ANGLE/src/tests/test_utils/ShaderExtensionTest.h:
(sh::ShaderExtensionTest::TestShaderCompile):
* Source/ThirdParty/ANGLE/util/autogen/angle_features_autogen.cpp:
* Source/ThirdParty/ANGLE/util/autogen/angle_features_autogen.h:
* Source/ThirdParty/ANGLE/util/capture/trace_fixture.cpp:
(SetFramebufferID):
(SetBufferID):
(SetRenderbufferID):
(SetTextureID):
(SetResourceID): Deleted.

Canonical link: https://commits.webkit.org/270363@main
